mocks/mocks.py

```diff
@@ -68,6 +68,42 @@ def crop_simulation_data(particle_pos,
     return pos, mass
 
 
+def init_mocks(config):
+    """
+    Creates some initials for the code. This function will be converted to
+    the __init__ when the code uses class.
+
+    Parameters
+    ----------
+    config : dict
+        A dictionary of configuration items.
+
+    Returns
+    -------
+    config : dict
+        A dictionary of configuration items.
+    telescope : gunagala.imager.Imager
+        Imager instance from gunagala.
+    psf_data : numpy.ndarray
+        Imager's psf data.
+    cosmo : astropy.cosmology.core.FlatLambdaCDM
+        The main cosmology that will be used by the code.
+    redshift : numpy.float
+        The redshift of the galaxy computed with respect to the distance of
+        the observer to the galaxy.
+    """
+    cosmo = create_cosmology(config)
+    redshift = compute_redshift(config['galaxy_distance'],
+                                cosmo)
+    config['pixel_scale'] = compute_pixel_scale(redshift,
+                                                cosmo,
+                                                config['sim_pc_pixel'])
+    imagers = gunagala.imager.create_imagers()
+    telescope = imagers[config['imager']]
+    psf_data = telescope.psf.pixellated()
```

Comment:
Suggest *not* returning psf_data as you then have two versions of the PSF in the notebook: `telescope.psf.pixellated()` and `psf_data`. Better to rely on gunagala class structure, so that someone can simply do:
```python
convolved = mocks.convolve_image_psf(noiseless,
                                    telescope.psf.pixellated(),
                                    convolution_boundary='extend')
```

Reply:
@lspitler 
It absolutely makes sense what you suggested and I applied it.